homeassistant/components/octoprint/manifest.json

```diff
@@ -1,7 +1,14 @@
 {
   "domain": "octoprint",
-  "name": "OctoPrint",
+  "name": "Octoprint",
+  "config_flow": true,
   "documentation": "https://www.home-assistant.io/integrations/octoprint",
-  "after_dependencies": ["discovery"],
-  "codeowners": []
-}
+  "requirements": [],
+  "ssdp": [],
+  "zeroconf": [],
```

Comment:
OctoPrint has zeroconf, and exposes a Unique ID via this route as well. This has been added in the more recent version of OctoPrint.

Reply:
We will need to migrate the legacy discovery as well

Reply:
Are there any docs on getting zeroconf and discovery working in the dev container? I am having trouble getting the mdns messages to the preview task.